arrows/vtk/applets/tests/CMakeLists.txt

```diff
@@ -0,0 +1,22 @@
+include(kwiver-test-setup)
+
+add_test(NAME vtk:applets:estimate_depth
+  COMMAND kwiver estimate-depth
+            --config "${kwiver_test_data_directory}/estimate_depth_testing.conf"
+            --video-source "${kwiver_test_data_directory}/aphill_240_1fps_crf32.mp4"
+            --input-cameras-dir "${kwiver_test_data_directory}/krtd"
+            --input-landmarks-file "${kwiver_test_data_directory}/landmarks.ply")
+add_test(NAME vtk:applets:fuse_depth
+  COMMAND kwiver fuse-depth
+            --config "${kwiver_test_data_directory}/fuse_depth_testing.conf"
+            --input-cameras-dir "${kwiver_test_data_directory}/krtd"
+            --input-landmarks-file "${kwiver_test_data_directory}/landmarks.ply"
+            --input-geo-origin-file "${kwiver_test_data_directory}/geo_origin.txt"
+            --input-depths-dir "${kwiver_test_data_directory}/depths")
+add_test(NAME vtk:applets:color_mesh
```

Comment:
This test is failing on some machines because of "bad X server connection".  I think this means we need to change how VTK is built on those systems.  Probably no change to this branch but maybe a change in Fletch or just a change in packages we install on the CI machines.  @chetnieter and @danlipsa know more about this.